scripts/fast-reboot

```diff
@@ -82,11 +86,16 @@ then
   systemctl stop nps-modules-`uname -r`.service
 fi
 
+# Update the reboot cause file to reflect that user issued 'fast-reboot' command
+# Upon next boot, the contents of this file will be used to determine the
+# cause of the previous reboot
+echo "User issued 'fast-reboot' command [User: ${REBOOT_USER}, Time: ${REBOOT_TIME}]" > ${REBOOT_CAUSE_FILE}
+
 # Wait until all buffers synced with disk
 sync
 sleep 1
 sync
 
 # Reboot: explicity call Linux native reboot under sbin
 echo "Rebooting to $NEXT_SONIC_IMAGE..."
-/sbin/reboot
+exec /sbin/reboot
```

Comment:
I'm not sure for this exec. Did you test it?

Reply:
Yes. I made the same change recently in the `scripts/reboot` script and tested that thoroughly.